test.sh

```diff
@@ -9,7 +9,9 @@ site_pkgs=$(python -c 'import site; print(site.getsitepackages()[0])')
 # Parse command line arguments
 for var in "$@"
 do
-    if [[ $var == "unit" ]]; then
+    if [[ $var == "check_notebooks" ]]; then
```

Comment:
Instead pf "check_notebooks" I would just call it "notebooks". So, from the command line it would be `./test.sh notebooks`

Reply:
I would add it after `coverage`